models/cex/ethereum/cex_ethereum_addresses.sql

```diff
@@ -0,0 +1,798 @@
+{{config(alias='addresses',
+        post_hook='{{ expose_spells(\'["ethereum"]\',
+                                    "sector",
+                                    "cex",
+                                    \'["hildobby", "soispoke", "web3_data"]\') }}')}}
+
+SELECT blockchain, LOWER(address) AS address, cex_name, distinct_name, added_by, added_date
+FROM (VALUES
+    -- Binance, Source: https://etherscan.io/accounts/label/binance
+    ('ethereum', '0x3f5ce5fbfe3e9af3971dd833d26ba9b5c936f0be', 'Binance', 'Binance 1', 'hildobby', timestamp('2022-08-28'))
+    , ('ethereum', '0xd551234ae421e3bcba99a0da6d736074f22192ff', 'Binance', 'Binance 2', 'hildobby', timestamp('2022-08-28'))
+    , ('ethereum', '0x564286362092d8e7936f0549571a803b203aaced', 'Binance', 'Binance 3', 'hildobby', timestamp('2022-08-28'))
+    , ('ethereum', '0x0681d8db095565fe8a346fa0277bffde9c0edbbf', 'Binance', 'Binance 4', 'hildobby', timestamp('2022-08-28'))
+    , ('ethereum', '0xfe9e8709d3215310075d67e3ed32a380ccf451c8', 'Binance', 'Binance 5', 'hildobby', timestamp('2022-08-28'))
+    , ('ethereum', '0x4e9ce36e442e55ecd9025b9a6e0d88485d628a67', 'Binance', 'Binance 6', 'hildobby', timestamp('2022-08-28'))
+    , ('ethereum', '0xbe0eb53f46cd790cd13851d5eff43d12404d33e8', 'Binance', 'Binance 7', 'hildobby', timestamp('2022-08-28'))
+    , ('ethereum', '0xf977814e90da44bfa03b6295a0616a897441acec', 'Binance', 'Binance 8', 'hildobby', timestamp('2022-08-28'))
```

Comment:
This line may be repeated with line 54, @hildobby you may need to choose to delete one of them.